pkg/git/common/common.go

```diff
@@ -28,6 +28,7 @@ const (
 	DefaultOrigin            = "origin"
 	DefaultBranch            = "master"
 	DefaultPublicKeyAuthUser = "git"
+	CAFile                   = "caFile"
```

Comment:
(**NB:** not specifically related to this line // note to self)

Looking at this with a fresh pair of eyes, I think the more (Go) idiomatic way of structuring the packages is to move all of `common` to `git`.

Can pick this up in #270.

Reply:
Let's do this in the refactoring PR.

Reply:
Already taken care of there :-)